pkg/reconciler/stats.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"github.com/knative/pkg/controller"
+	"go.uber.org/zap"
+)
+
+// MustNewStatsReporter creates a new instance of StatsReporter. Panics if creation fails.
+func MustNewStatsReporter(reconciler string, logger *zap.SugaredLogger) controller.StatsReporter {
```

Comment:
I don't think we need this, this is used in Serving to collect stats, but we don't have this yet in pipeline

Reply:
Think for telling me that, I will remove it.